README.md

```diff
@@ -32,36 +32,24 @@ python gen_confs.py
 
 ### L1 setup
 
+Run `./build-l1-geth.sh` or,
+
 ```shell
 # install upstream geth:
 go install github.com/ethereum/go-ethereum/cmd/geth@v1.10.15
 
 # Create L1 data dir
 geth init --datadir data_l1 l1_genesis.json
 
-# Run L1 geth
-geth --datadir data_l1 \
-    --networkid 900 \
-    --http --http.api "net,eth,consensus" \
-    --http.port 8545 \
-    --http.addr 127.0.0.1 \
-    --http.corsdomain "*" \
-    --ws --ws.api "net,eth,consensus" \
-    --ws.port=8546 \
-    --ws.addr 0.0.0.0 \
-    --maxpeers=0 \
-    --vmodule=rpc=5
-
-# Get the genesis block hash (while running the above command)
-curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["0x0", false],"id":1}' http://localhost:8545 | jq -r ".result.hash" | tee l1_genesis_hash.txt
```

Comment:
There was no need to do this before importing the signer account, so I moved this step towards the end (and into `get-genesis-hashes.sh`.

Reply:
The idea was to get the genesis hash before creating lots of blocks with the mine flag, to give a human some time to start the other things and have rollup genesis at L1 genesis. With automation of the process it doesn't really matter though, great!